src/main/java/seedu/address/model/property/client/Client.java

```diff
@@ -97,25 +126,29 @@ public String toString() {
         final StringBuilder builder = new StringBuilder();
 
         if (clientName != null) {
-            builder.append("Client Name: ").append(getClientName());
+            builder.append(CLIENT_NAME).append(getClientName());
         }
         if (clientContact != null) {
             if (builder.length() != 0) {
                 builder.append("; ");
             }
-            builder.append("Client Contact: ").append(getClientContact());
+            builder.append(CLIENT_CONTACT).append(getClientContact());
         }
         if (clientEmail != null) {
             if (builder.length() != 0) {
                 builder.append("; ");
             }
-            builder.append("Client Email: ").append(getClientEmail());
+            builder.append(CLIENT_EMAIL).append(getClientEmail());
         }
         if (clientAskingPrice != null) {
             if (builder.length() != 0) {
                 builder.append("; ");
             }
-            builder.append("Client Asking Price: ").append(getClientAskingPrice());
+            builder.append(CLIENT_PRICE).append(getClientAskingPrice());
+        }
+
+        if (builder.length() != 0) {
+            builder.append(";");
         }
```

Comment:
There will be an extra `;` in the `toString` of `Property` when both the client information and tags are printed
E.g. `Client Asking Price: $800,00;; Tags: [4 bedrooms]`